src/components/MicroblockDetailsPanel.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <app-panel class="microblock-details-panel">
+    <header class="microblock-details-panel__header">
+      <h2 class="microblock-details-panel__heading h3">
+        DETAILS
+      </h2>
+      <div>
+        <copy-chip
+          :label="microblockDetails.hash"
+          class="microblock-details-panel__chip"/>
+        <copy-chip
+          :label="formatEllipseHash(microblockDetails.hash)"
+          :clipboard-text="microblockDetails.hash"
+          class="microblock-details-panel__chip-ellipse"/>
+      </div>
+    </header>
+    <table>
+      <tbody>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            Keyblock
+          </th>
+          <td class="microblock-details-panel__data">
+            <app-link :to="`/microblocks/${microblockDetails.prev_key_hash}`">
+              <span class="microblock-details-panel__hash">
+                {{ microblockDetails.prev_key_hash }}
+              </span>
+              <span class="microblock-details-panel__hash-ellipse">
+                {{ formatEllipseHash(microblockDetails.prev_key_hash) }}
+              </span>
+            </app-link>
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            Microblock Height
+          </th>
+          <td class="microblock-details-panel__data">
+            {{ formatNumber(microblockDetails.height) }}
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            Time
+          </th>
+          <td class="microblock-details-panel__data">
+            <datetime-label :datetime="microblockDetails.time"/>
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            Transactions Count
+          </th>
+          <td class="microblock-details-panel__data">
+            {{ formatNumber(microblockDetails.transactions_count) }}
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            API links
```

Comment:
```suggestion
            API Links
```

Reply:
Good idea, applied here in all other places where it was also wrong